docs/README.md

```diff
@@ -118,22 +118,27 @@ Booster's ultimate goal is making developer's lives easier, fulfilling the dream
 
 ### Booster Principles
 
-Booster takes a holistic and highly-opinionated approach at many levels:
+Booster boosts developers's productivity by focusing only on business logic. Write your business logic and provide your credentials. Let Booster do the rest. Booster takes a holistic and highly-opinionated approach at many levels:
 
 - **Focus on business value**: The only code that makes sense is the code that makes your application different from any other.
 - **Convention over configuration**: All the supporting code and configuration that is similar in all applications should be out of programmers’ sight.
-- **Serverless-less**: Why go Serverless to avoid managing infrastructure when you can implicitly infer your Serverless architecture from your code and not even deal with that?
+- **Truly Serverless**: Why go Serverless to avoid managing infrastructure when you can implicitly infer your Serverless architecture from your code and not even deal with that?
+- **Effective Multicloud**: Booster design makes it possible to run the same application in AWS, Azure or Kubernetes clusters with no code changes in your application.
 - **Scale smoothly**: The code you write to handle your first 100 users will still work to handle your first million. You won't need to rewrite your application when it succeeds.
 - **Event-source and CQRS**: Our world is event-driven, businesses are event-driven, and modern software maps better to reality when it’s event-driven.
 - **Principle of Abstraction**: Building an application is hard enough to have to deal with recurring low-level details like SQL, API design, or authentication mechanisms, so we tend to build more semantic abstractions on top of them.
 - **Real-time first**: Client applications must be able to react to events happening in the backend and notice data changes.
+- **Extensible**: Rockets are what we call plugins in the Booster ecosystem. A rocket is a regular node package that works out of the box and provides new end-to-end abstractions, supports new cloud services, or pre-built functionalities that you can install in your project.
 
 ### Why use Booster
 
+What does Booster boost? Your team’s productivity. Not just because it makes you work faster, but because it makes you worry about less buttons and switches.  We aim to solve major productivity sinks for developers like designing the right cloud infrastructure, writing APIs or dealing with ORMs.
+
```

Comment:
```suggestion
What does Booster boost? Your team’s productivity. Not just because it makes you work faster, but because it makes you worry about fewer buttons and switches. We aim to solve major productivity sinks for developers like designing the right cloud infrastructure, writing APIs or dealing with ORMs.

```

Reply:
Not sure about this change so don't give it much attention 😛 : "less" -> "fewer"